### ISSUE TITLE
InteractiveUtils not available in integrated REPL

### ISSUE THREAD

**Issue by pfitzseb** [julia-vscode/julia-vscode#3648]
This is on 1.102.2:
```
[ Info: Precompiling VSCodeServer [9f5989ce-84fe-42d4-91ec-6a7a8d53ed0f]
julia> varinfo()
ERROR: UndefVarError: `varinfo` not defined
Stacktrace:
 [1] top-level scope
   @ REPL[1]:1

julia> InteractiveUtils
InteractiveUtils

julia> InteractiveUtils.varinfo()
  name                         size summary                                                    
  ––––––––––––––––––––––––– ––––––– –––––––––––––––––––––––––––––––––––––––––––––––––––––––––––
  Base                              Module                                                     
  Core                              Module                                                     
  Main                              Module                                                     
  global_err_handler        0 bytes global_err_handler (generic function with 1 method)        
  is_disconnected_exception 0 bytes is_disconnected_exception (generic function with 4 methods)
```

**Comment by davidanthoff** [comment]
Oh, I thought I had introduced that error in https://github.com/julia-vscode/julia-vscode/pull/3629, but apparently it crept in earlier... Might make sense to just fix this as part of #3629, right?

**Comment by davidanthoff** [comment]
Ok, I think I know what is causing this and will fix it as part of #3629.
